### DISCUSSION TITLE
Enemies spawning and attacking the sub immediately upon loading level

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13614] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Threshers spawned immediately upon loading the level in multiplayer (unstable)

Normally monster events only spawn after the submarine has moved 50m? (`mindistancetravelled`, `allowatstart` ?)

[Season_5.5.zip](https://github.com/FakeFishGames/Barotrauma/files/14625456/Season_5.5.zip)

### Reproduction steps

Load the save

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
I unfortunately couldn't find a way to repro this. :/ Not seeing any thresher spawns at the start of the round in this save, it seems to take several minutes for them to appear.

**Comment by ubertpendragon** [comment]
https://cdn.discordapp.com/attachments/668819422919524352/1219311979131310170/hammerheadjumpscare.mp4?ex=660ad7c5&is=65f862c5&hm=f6b54abcc238967aac9aae2143ffb54e389ae812694367bdadb608b41edf438d&

One hypothesis is great sea vertical sub-biome spawning the sub far enough to trigger the events

**Comment by Wheatley1001** [comment]
This is an inconsistent bug that has occurred throughout the 1.2.13.0 test campaign (which the save file comes from). There are a number of observed issues with the event manager that compliment this behaviour, although they deserve their own tickets if they do not already exist. Countless times throughout our test campaign we have been swarmed by enemies before everyone had loaded in. I, the captain, even loaded into the game as a corpse because I was killed by water pressure before I was given a chance to respond (which I will expand on in a different report, since _that_ is a different issue).

**Comment by SomeRandomNoobKekeke** [comment]
I found a way to reproduce:
This is happening when you're passing uninhabited spot with some monsters on your tail
![uninhabited](https://github.com/FakeFishGames/Barotrauma/assets/122838333/a32c8a48-1164-4467-bc6d-027fb6e402ad)
1. travel from outpost to empty spot
2. teleport the sub around the level, make sure something spawned, but don't kill it
3. teleport to end, transit to some next location 
4. take a look around, there's 100% something prespawned. Sometimes it's just a few crawlers right outside the sonar radius, sometimes it's 10 hammerheads on top of your sub

If you retry or save-load nothing will spawn
It can happen when transitioning from outpost to outpost, but i don't know exact conditions for that

**Comment by Regalis11** [comment]
[I've created an issue ticket](https://github.com/FakeFishGames/Barotrauma/issues/13750) about this in the issue tracker, we'll look into it! Sorry for the inconvenience.

**Comment by Regalis11** [comment]
This has been fixed in the latest hotfix, closing. PR in our private repo: https://github.com/FakeFishGames/Barotrauma-development/pull/5266
